tests/test_transformer.py

```diff
@@ -792,3 +792,15 @@ def test_transformer__RestrictingNodeTransformer__visit_Import(compile):
     code, errors = compile('from a import _m as n')[:2]
     assert code == None
     assert errors[0] == (errmsg % '_m')
+
+
+@pytest.mark.parametrize(*compile)
+def test_transformer__RestrictingNodeTransformer__visit_ClassDef(compile):
+    code, errors = compile('class Good: pass')[:2]
```

Comment:
It would be nice to have a docstring telling the goal of the test. I could think of something like `"""It forbids class names starting with an underscore."""`